agent/config/config.go

```diff
@@ -218,7 +219,8 @@ func fileConfig() (Config, error) {
 
 	err = json.Unmarshal(data, &cfg)
 	if err != nil {
-		seelog.Errorf("Error reading cfg json data, err %v", err)
+		seelog.Criticalf("Error reading cfg json data, err %v", err)
```

Comment:
Is it worth to logging the `string(data)` here?

Reply:
Yeah, not a bad idea.

Reply:
After further thought, I'm going to leave this out for now.  I'm concerned about the case where a customer adds credentials (`EngineAuthData`) to the file and then we have a parse error; that would cause credentials to be written out to the log.